app/controllers/application_controller.rb

```diff
@@ -0,0 +1,21 @@
+require 'pry'
+require './config/environment'
+
+class ApplicationController < Sinatra::Base
+
+  get "/api/v1/weather" do
+    # params = {"lat"=>"35", "lon"=>"139"}
+    lat = params[:lat]
```

Comment:
for my understanding: the lat and long params, will be provided from the flight API, sent to the BE, the BE will then take those and make the call to the weather API. Is that right?

Reply:
yes... these changes have been made, but we are pushing up those changes...